.github/workflows/python-cross-platform-support.yml

```diff
@@ -87,8 +88,23 @@ jobs:
           cd (Get-ChildItem -Filter aac_secure_install_*)
           .\install_aac_air_gap.bat
 
-      - name: Test Plugin Commands
-        run: echo "Replace with testing the package commands of this repo"
+      - name: Get Package Version
+        run: python -m aac version
+
+      - name: Check AaC DSL
+        if: matrix.os != 'windows-latest'
+        run: |
+          spec_file=$(find $pythonLocation -name aac.aac | head -n1)
+          python -m aac check $spec_file
+
+      - name: Check AaC DSL
+        if: matrix.os == 'windows-latest'
+        run: |
+          $spec_file=$(gci -r -fi aac.aac -Name "$Env:pythonLocation")
+          python -m aac check "$Env:pythonLocation\$spec_file"
+
+      - name: Print out Core Spec
+        run: python -m aac print-defs
```

Comment:
Looks like we messed up in the template, it should have been the genericized about `Test Plugin Commands` and these should only be used in the core aac repo where we can exercise those commands and the plugins should test their commands.